enterprise/admin/views.py

```diff
@@ -205,11 +205,13 @@ def _handle_singular(cls, enterprise_customer, manage_learners_form):
         form_field_value = manage_learners_form.cleaned_data[ManageLearnersForm.Fields.EMAIL_OR_USERNAME]
         email = email_or_username__to__email(form_field_value)
         try:
-            validate_email_to_link(email, form_field_value, ValidationMessages.INVALID_EMAIL_OR_USERNAME)
+            validate_email_to_link(email, form_field_value, ValidationMessages.INVALID_EMAIL_OR_USERNAME, True)
```

Comment:
@haikuginger @e-kolpakov Do either of you know what the purpose of the `ignore_existing` kwarg is/was on the [validate_email_to_link](https://github.com/edx/edx-enterprise/blame/master/enterprise/admin/utils.py#L131) function? I can't think of a reason to ever need to raise an exception if there is an existing `EnterpriseCustomerUser`.

Reply:
@douglashall If I recall right at that point the decision was that customer can only be linked to single enterprise customer, so this is a mechanism to indicate that a student is already linked to enterprise customer.

In the original PR, it was set to ignore bulk uploads (for the purposes of allowing same file loaded multiple times), but report it during adding single student. At that point it was considered as malformed request and reported as validation error.